package.json

```diff
@@ -4,7 +4,32 @@
     "jasmine-core": "^5.1.1"
   },
   "scripts": {
+    "lint:fix": "standardx --fix",
+    "lint:js": "standardx 'spec/javascripts/**/*.js' 'src/**/*.js'",
     "test": "jasmine-browser-runner runSpecs",
     "build": "bash build.sh"
+  },
+  "dependencies": {
+    "standardx": "^7.0.0"
+  },
+  "eslintConfig": {
+    "env": {
+      "browser": true,
+      "jasmine": true
+    },
+    "rules":{
+      "no-var":0,
+      "no-unused-vars" : 0,
+      "no-use-before-define": 0
+    }
```

Comment:
> Small non-blocking suggestion, the spacing for each rule is slightly different, we could keep it the same as other key value pairs in this file.

Yep, good suggestion @MartinJJones.

> It may have already been considered, but just in case, it appears we can also create a .eslintrc file in the root directory and move the eslintConfig rules here - https://www.npmjs.com/package/standardx#overview, for example:

I don't expect we'll need many more rules so I think it's probably okay to add the config rules in `package.json` for the time being which aligns with how other GOV.UK apps are setup e.g. [govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components/blob/main/package.json#L21-L25) and [government-frontend](https://github.com/alphagov/government-frontend/blob/main/package.json#L28-L32)

Reply:
Thanks @jon-kirwan, yeah probably don't need the separate `.eslintrc` file in that case, also good to keep it consistent with other GOV.UK apps, @kashifatcha think it's just the spacing issue to address here, I should have added that on a separate message to make it easier to resolve the conversation.